members/views.py

```diff
@@ -45,3 +45,24 @@ def post(self, request):
             return JsonResponse({"message": "SUCCESS"}, status=201)
         except Exception:
             return JsonResponse({"message": "INVALID_VALUE"}, status=400)
+
+class SignInView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            if not (data.get('member') or data.get('password')):
+                return JsonResponse({"message": "EMPTY_VALUE"}, status=400)
+            
+            member = data['member'].replace("-", "")
+            
+            if not Member.objects.filter(Q(name=member) | Q(phone_number=member)).exists():
+                return JsonResponse({"message": "INVALID_MEMBER"}, status=401)
+            if not bcrypt.checkpw(data['password'].encode('utf-8'), Member.objects.get(Q(name=login_name) | Q(phone_number=login_phone_number)).password.encode('utf-8')):
+                return JsonResponse({"message": "INVALID_MEMBER"}, status=401)
+            
+            token = jwt.encode({'id': Member.objects.get(Q(name=login_name) | Q(phone_number=login_phone_number)).id}, SECRET_KEY, algorithm='HS256')
```

Comment:
```suggestion
            q= Q(name=login_name) | Q(phone_number=login_phone_number)
            ...
            token = jwt.encode({'id': Member.objects.get(q).id}, SECRET_KEY, algorithm='HS256')
```
Q을 사용한 부분이 반복되고 길어서 가독성이 떨어집니다. 조건(Q)에 대해서 미리 선언한 뒤에 사용하면 좋을 것 같아요.

Reply:
넵 수정하겠습니다.